lib/huginn_scheduler.rb

```diff
@@ -169,6 +169,7 @@ def cleanup_expired_events!
   end
 
   def cleanup_failed_jobs!
+    return if Rails.configuration.active_job.queue_adapter != :delayed_job
```

Comment:
[`sidekiq-failures`](https://github.com/mhfs/sidekiq-failures#maximum-tracked-failures) takes care of this, its configured in the initializer.